### DISCUSSION TITLE
Not able to render Font Family while conversion to PNG & JPG

### DISCUSSION THREAD

**Discussion by abhilab** [vvvv/SVG#1060] [Q&A]
**Description**
I added Font Face to SVG, Font family shows in SVG Preview, but when I convert SVG to Bitmap using Draw method, then save the Bitmap to PNG, JPG. It does not render the font.
NOTE: The font shows on SVG but not in PNG, JPG

**Example data
Code in SVG:**

<style>
https://github.com/font-face {
font-family:Segoe Pro Bold;
src: url("data:font/truetype;charset=utf-8;base64, my fonts base64 code") format('truetype');
}
</style>
**Code in C#**
Bitmap bmp = mySvg.Draw(pixelWidth, pixelHeight);
bmp.Save(@"Filename.jpeg", jgpEncoder,myEncoderParameters);

**Used Versions**
SVG version used: **3.4.4**

_Please help me out, I am stuck at the end of deadline, all my work is complete for image generation, only the Font family is not showing. Thanks in advance._

**Comment by H1Gdev** [comment]
SVG is fragmentary so I don't know how you want it to be styled...

It is possible to apply fonts using PrivateFont.
Ref. https://github.com/svg-net/SVG/blob/master/Tests/Svg.UnitTests/PrivateFontsTests.cs

**Reply by H1Gdev** [reply]
```cs
SvgFontManager.PrivateFontPathList.Add(@"path\to\fontfile");
SvgFontManager.LocalizedFamilyNames.Add(new string[] { "Yumin", "游明朝", }); // if needed.
```
